kernelci.org/content/en/docs/tests/_index.md

```diff
@@ -24,5 +24,8 @@ Non-native tests are run in fully autonomous systems, such as syzbot or CKI.
 Their results are shared alongside KernelCI native test results via
 [KCIDB](https://kcidb.kernelci.org).
 
+It's possible for anyone to add new external tests.  See the [How-To
+External guide](howto_external) to get started.
```

Comment:
maybe is "(how_to_external)" ?

Reply:
i was just mimicing the howto.md name.  Happy to change it if having the names look different are ok.

Reply:
ok, if original is same works also for me